lab_7_llm/settings.json

```diff
@@ -1,8 +1,8 @@
 {
   "parameters": {
-    "model": "",
-    "dataset": "",
-    "metrics": []
+    "model": "cointegrated/rubert-base-cased-nli-threeway",
+    "dataset": "xnli",
+    "metrics": ["accuracy"]
   },
-  "target_score": 0
+  "target_score": 4
```

Comment:
I think it should change to 6?

Reply:
Yes, I'll change it when I'll complete all tasks for this mark